cohttp-async/src/server.mli

```diff
@@ -16,9 +16,16 @@ type 'r respond_t =
   ?body:Body.t ->
   Cohttp.Code.status_code -> 'r Async_kernel.Deferred.t
 
+(** A request handler can respond in two ways:
+    - Using [`Response], with a [Response.t] and a [Body.t].
```

Comment:
Minor: these `[Response.t]` can be `{!Response.t}` and they will be hyperlinked by odoc (useful in the functor heavy world of cohttp :)

Reply:
I updated all of the references now -- I don't know whether you meant only `[Response.t]`..

Reply:
All the references are great; it's useful for anything in the ocamldoc comment that is a linkable value, to give odoc more metadata to work with.